### DISCUSSION TITLE
Change scene item value using web socket

### DISCUSSION THREAD

**Discussion by Tadejo9** [obs-websocket-community-projects/obs-websocket-js#326] [Q&A]
Hello,

is it currently possible to change value of Text item using websocket?

User clicks a button in application and value of text in obs is updated to value sent via websocket?

**Comment by Mattyx00** [comment]
Hello @Tadejo9 any news? I would like to do the same.
thanks.

**Comment by AndiLeni** [comment]
@Tadejo9 @Mattyx00 
I needed this too and this script worked for me. If anyone is searching for how to do this in the future:

```js
obs.call("SetInputSettings", { inputName: "RunningTime", inputSettings: { text: runningTime } }); 
```

RunningTime is the name of my source in OBS:
![image](https://github.com/obs-websocket-community-projects/obs-websocket-js/assets/13385771/fea2a987-85ea-4e24-93e7-4e147ef42e30)
